plugins/modules/instance.py

```diff
@@ -83,6 +83,12 @@
         type=FieldType.string,
         description=["The filesystem to create this disk with."],
     ),
+    "disk_encryption": SpecField(
+        type=FieldType.string,
+        description="The disk encryption status of this disk."
+        + "NOTE: Disk encryption may not currently be available to all users.",
+        choices=["enabled", "disabled", "pde"],
+    ),
```

Comment:
In the sepc doc this field seems only accept two values: `disabled`,`enabled`. And we probably can set its default to be `enabled`.

Reply:
The API spec mentioned `pde` as one of the possible values in the "Schema" section. Initially I had it set a default value but this was causing some problems because the API would recognize creating an instance without specifying the `disk_encryption` status as an update.